release-scripts/prepublish.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/bash
+
+libNames=("msal-core" "msal-common" "msal-browser" "msal-node" "msal-angular" "msal-react" "msal-node-extensions");
+
+declare -A publishFlagNames;
+
+publishFlagNames["msal-core"]=publishMsalCore;
+publishFlagNames["msal-common"]=publishMsalCommon;
+publishFlagNames["msal-browser"]=publishMsalBrowser;
+publishFlagNames["msal-node"]=publishMsalNode;
+publishFlagNames["msal-angular"]=publishMsalAngular;
+publishFlagNames["msal-react"]=publishMsalReact;
+publishFlagNames["msal-node-extensions"]=publishMsalNodeExtensions;
+
+# Iterate each library directory name
+for i in "${libNames[@]}"; do
+    libPath="../lib/${i}/package.json"
```

Comment:
Note that msal-node-extensions is not in the `lib` folder.

Reply:
Good catch, I missed it.